Resources/Regression/qml/RegressionLinear.qml

```diff
@@ -38,7 +38,7 @@ Form
 				{ label: qsTr("Stepwise"),	value: "stepwise"}
 			]			
 		}
-		AssignedVariablesList { name: "covariates";	title: qsTr("Covariates");			suggestedColumns: ["scale"]								}
+		AssignedVariablesList { name: "covariates";	title: qsTr("Covariates");			allowedColumns: ["scale"]								}
```

Comment:
Should this also be done for the Bayesian one?

Reply:
I think the Bayesian regression already disallows factors.

Reply:
Ok, perfect